Add PeerTableQmlModel

johnny9 opened the pull request:
The upstream PeerTableModel does not expose its startAutoRefresh and stopAutoRefresh methods as invokable so the PeerTableQmlModel is introduced to make these methods invokable in qml.

fixes #482

johnny9 committed:
Add PeerTableQmlModel

The upstream PeerTableModel does not expose its startAutoRefresh
and stopAutoRefresh methods as invokable so the PeerTableQmlModel
is introduced to make these methods invokable in qml.

hebasto commented on qml/models/peertableqmlmodel.h:
```diff
@@ -0,0 +1,23 @@
+    Q_INVOKABLE void stopAutoRefresh();
+};
+
+#endif // PEERTABLEQMLMODEL_H
```
```suggestion
#endif // BITCOIN_QML_MODELS_PEERTABLEQMLMODEL_H
```

johnny9 commented:
Update from [14c2540](https://github.com/bitcoin-core/gui-qml/commit/14c25409afce43ace9172c73feb3be1c714a4925) to [a0d6179](https://github.com/bitcoin-core/gui-qml/commit/a0d617925d17e4af15e1b424e72c9c6fbbd71671)
 - Fixed include guard comment in peertableqmlmodel.h
 - Added copyright comment to the top of the new files

hebasto reviewed (approved):
ACK a0d617925d17e4af15e1b424e72c9c6fbbd71671, tested on Ubuntu 24.04.
